docs/intro/installation.md

```diff
@@ -0,0 +1,359 @@
+
+### Create a file system wallet
+
+To deploy a program with the Solana CLI, you will need a Solana wallet with SOL
+tokens to pay for the cost of transactions and data storage on the blockchain.
+
+To create a file system wallet to use during local development, run the
+following command:
+
+```shell
+solana-keygen new
+```
+
+By default, the `solana-keygen` command will create a new file system wallet
+located at `~/.config/solana/id.json`. You can manually specify the output file
+location using the `--outfile /path` option.
+
+<Callout type="note">
+
+If you already have a file system wallet saved at the default location, this
+command will **NOT** override it unless you explicitly force override using the
+`--force` flag.
+
+</Callout>
+
+With your new file system wallet created, you must tell the Solana CLI to use
+this wallet to deploy and take ownership of your on-chain program:
+
+```shell
+solana config set -k ~/.config/solana/id.json
+```
+
+You can view your current Solana CLI configuration settings with the following
+command:
+
+```shell
+solana config get
+```
+
+### Airdrop SOL tokens to your wallet
+
+Once you've set up your local wallet, request an airdrop of SOL to fund your
+wallet. You need SOL to pay for transaction fees and to deploy programs.
+
+Set your cluster to the devnet:
+
+```shell
+solana config set -ud
+```
+
+Then request an airdrop of devnet SOL:
+
+```shell
+solana airdrop 2
+```
+
+You can check your current wallet's SOL balance any time:
+
+```shell
+solana balance
+```
+
+<Callout>
+
+The `solana airdrop` command has a limit on how many SOL tokens can be requested
+_per airdrop_ for each cluster (testnet or devnet). If your airdrop transaction
+fails, lower your airdrop request quantity and try again. You can also get your
+wallet address using `solana address` and request testing sol on the
+[Solana Web Faucet](https://faucet.solana.com). If you ever need bigger amounts
+you can find ways to do so in this
+[airdrop guide](https://solana.com/de/developers/guides/getstarted/solana-token-airdrop-and-faucets)
+
+</Callout>
+
+### Setup a localhost blockchain cluster
+
+The Solana CLI comes with the
+[test validator](https://docs.solanalabs.com/cli/examples/test-validator)
+built-in. Running a local validator will allow you to deploy and test your
+programs locally.
+
+In a separate terminal, run the following command to start a local validator:
+
+```shell
+solana-test-validator
+```
+
+<Callout>
+
+In WSL you may need to first navigate to a folder where you have default write
+access:
+
+```shell
+cd ~
+mkdir validator
+cd validator
+solana-test-validator
+```
+
+</Callout>
+
+Then update the Solana CLI config to use the local validator:
+
+```shell
+solana config set -ul
+```
+
+To update the Solana CLI config to use a different cluster, you can use the
+following commands:
+
+```shell
+solana config set -ud    # For devnet
+solana config set -um    # For mainnet-beta
+solana config set -ut    # For testnet
```

Comment:
Rather than commands plus a comment to help understand the command, it might be easier to understand to use the full command. 
```suggestion
solana config set -u devnet
solana config set -u mainnet-beta
solana config set -u testnet
```

Reply:
added full commands:
```
solana config set --url mainnet-beta
solana config set --url devnet
solana config set --url localhost
solana config set --url testnet
```